Source/EventFlow.TestHelpers/MsSql/MsSqlHelpz.cs

```diff
@@ -50,7 +50,8 @@ public static MsSqlConnectionString CreateConnectionString(string label)
                 {
                     DataSource = FirstNonEmpty(
                         Environment.GetEnvironmentVariable("EVENTFLOW_MSSQL_SERVER"),
-                        ".")
+                        "."),
+                    Encrypt = false
```

Comment:
[Trust Server Cert](https://learn.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlconnectionstringbuilder.trustservercertificate?view=dotnet-plat-ext-8.0) wasn't working (I assume with SQL Server 2017), so I just disabled encryption. It's not required for tests.